src/platform/lumin-runtime/controllers/builders/controller-builder.js

```diff
@@ -84,7 +84,7 @@ export class ControllerBuilder {
 
     _updateRootNodeProperties(controller, properties) {
         const root = controller.getRoot();
-        console.log(JSON.stringify(root));
+        // console.log(JSON.stringify(root));
```

Comment:
remove the lines, don't comment them out

Reply:
👍